### ISSUE TITLE
getter-return: does not check Reflect.defineProperty

### ISSUE THREAD

**Issue by mfulton26** [denoland/deno_lint#1073]
### Lint Name

getter-return

### Code Snippet

```ts
Object.defineProperty({}, "red", { get: () => {} });
Reflect.defineProperty({}, "green", { get: () => {} });

```

### Expected Result

warnings on both getters which have no return values

### Actual Result

only a warning on `Object.defineProperty` usage but not on `Reflect.defineProperty`

### Additional Info

`Reflect.defineProperty` is like `Object.defineProperty` but returns a `boolean`.

### Version

deno 1.25.0 (release, aarch64-apple-darwin)
v8 10.6.194.5
typescript 4.7.4

**Comment by mfulton26** [comment]
same goes for `Object.create`; I can open a separate issue for such if that is preferred
